src/applications/mne_analyze/libs/anShared/CMakeLists.txt

```diff
@@ -90,6 +88,12 @@ target_link_libraries(${PROJECT_NAME} PRIVATE
     mne_inverse
     mne_rtprocessing
     mne_events
+)
+
+target_link_libraries(${PROJECT_NAME} PRIVATE
+    ${QT_REQUIRED_COMPONENT_LIBS}
+    ${MNE_LIBS_REQUIRED}
+    eigen
     ${FFTW_LIBS})
```

Comment:
I've just learned that this QT_REQUIRED_COMPONENT_LIBS, while it is nice and I think it is ok to keep it, we kind of no longer need it. Qt people have made Qt libs from 5.15 on, version independent (you specify you link t core and cmake links to qt5 or qt6 depending on what you have installed, what find_package finds, etc...